src/components/ProfitAndLossDetailedCharts/ProfitAndLossDetailedCharts.tsx

```diff
@@ -66,6 +66,14 @@ export const ProfitAndLossDetailedCharts = ({
 
       <header className='Layer__profit-and-loss-detailed-charts__header--tablet'>
         <BackButton onClick={() => setSidebarScope(undefined)} />
+        <div className='Layer__profit-and-loss-detailed-charts__head'>
+          <Text size={TextSize.lg} weight={TextWeight.bold} className='title'>
```

Comment:
Should this className be just "title" or "date" ? Not like "Layer__title"?

Reply:
I would go with plain `title` and `date`. `Layer__title` indicates on more globally used component.